packaging/aznfs/DEBIAN/control

```diff
@@ -0,0 +1,9 @@
+Package: aznfs
+Version: x.y.z
+Priority: required
+Architecture: amd64
+Origin: Microsoft
+Maintainer: Azure Storage XNFSv3 IDC Team <xnfsv3devsidc@microsoft.com>
+Homepage: https://github.com/Azure/BlobNFS-mount/blob/main/README.md
+Description: Mount helper program for correctly handling endpoint IP address changes for Azure Blob NFS mounts.
+Depends: conntrack, iptables, systemd, bind9-host, iproute2, util-linux, nfs-common
```

Comment:
we cannot put a dependency on systemd since we cannot change the distro's init system if it uses a different one.
what best we can do is check for systemd in preinst checks and fail the install with an appropriate error if someone tries on a non-systemd system.
you can do the following check

init="$(ps -q 1 -o comm=)"
if [ "$init" != "systemd" ]; then
    echo "Cannot install this package on a non-systemd system!"
    exit 1
fi